apps/aehttp/src/aehttp_api_validate.erl

```diff
@@ -188,7 +188,12 @@ prepare_param_({"maximum", Max}, Value, Name, _) ->
     end.
 
 get_param_value("body", _, Req0) ->
-    case cowboy_req:read_body(Req0) of
+    %% Cowboy will attempt to read up to ~5MB of data for up to 10s. The call
+    %% will return when there is up to ~5MB of data or at the end of the 10s
+    %% period. If there is more data to read (after reading the initial 5MB),
+    %% {more, Body, Req} is returned and we can consider the request body to
+    %% be too big and the request is invalid.
+    case cowboy_req:read_body(Req0, #{length => 5000000, period => 10000}) of
```

Comment:
I see we use cowboy 2.2.2
I see [defaults documented as 8MB and 15 seconds](https://ninenines.eu/docs/en/cowboy/2.2/manual/cowboy_req.read_body/). Re units, I see code [8000000 (B) and 15000 (ms)](https://github.com/ninenines/cowboy/blob/2.2.2/src/cowboy_req.erl#L462-L463).
Ok.